README.md

```diff
@@ -270,6 +273,22 @@ ModuleNotFoundError: No module named 'hyperonpy'
 Please ensure you have installed the Python module, see
 [Running Python and MeTTa examples](#running-python-and-metta-examples).
 
+### Rust REPL cannot load Python library
+
+Rust REPL is not compatible with Python virtual environments like Conda and Pyenv
```

Comment:
```suggestion
The REPL needs a path to the libpython library in the current environment.  This can be done one of two ways:

#### On Linux
1. Use `patchelf` on resulting REPL binary to link it with `libpython.so`
...
2. Set the `LD_LIBRARY_PATH` environment variable
...

#### On Mac OS
1. Use `install_name_tool` to change the REPL binary's link path
` ` `
otool -L target/debug/metta-repl | grep libpython ; to find <libpython-name>
install_name_tool -change <libpython-name> <path-to-libpython-in-virtual-env> target/debug/metta-repl
` ` `
2. Set the `DYLD_FALLBACK_LIBRARY_PATH` environment variable
` ` `
export DYLD_FALLBACK_LIBRARY_PATH=<path-to-libpython-directory-in-virtual-env>
` ` `
This can be done in your `~/.bashrc` file so there is no need to do it each time you launch the REPL.
```